### ISSUE TITLE
Expected input shapes of the parameters

### ISSUE THREAD

**Issue by bezirganyan** [benedekrozemberczki/pytorch_geometric_temporal#131]
Hello, 
I am currently trying to develop a GNN architecture for my thesis, and what makes the process really hard is that  there is no indication what the expected shapes of the input parameters shall be. For example, I am currently using the MPNNLSTM module, and I have a hard time trying to make it work with a window other than one. I guessed that my input shall be `[window x node_count x feature_size]`, but now I am getting errors from batch_norm layers inside the MPNNLSTM. It would be great if you could add the expected shapes in the documentation and doc string, and in this particular case indicate what the input shapes of MPNNLSTM shall be. Thanks

**Comment by benedekrozemberczki** [comment]
How about you add it? Nice learning experience, becoming an open source contributor - it is pretty empowering.

**Comment by bezirganyan** [comment]
Thanks for the suggestion, and I gladly would add it, the problem is I am not nearly as familiar to the modules and architectures as the people who developed them. I opened the issue, as I believed that it would take not much time for the original developers to add the shapes, while I am still trying to figure out what the input shape of the MPNNLSTM has to be. If that's not that easy I will be happy to spend enough time and contribute back to this project after my thesis is finished in two months. 

At this point, as you are the main contributor of MPNNLSTM module, it would be an enormous help if you could elaborate briefly on what the shape of X shall be for the forward function. 
As mentioned above I did try `[window x node_count x feature_size]`, which is the only way I managed to pass the tensor reshaping line: 
```
S = X.view(-1, self.window, self.num_nodes, self.in_channels)
```
However, this shape doesn't pass the BatchNorm 1 layer. 
Can you please indicate if there is a bug for window >= 1, or if the input shape shall be different?

Thanks
